src/Page/GetHelp.elm

```diff
@@ -32,24 +34,77 @@ columnStyle =
         ]
 
 
-card : String -> String -> String -> Html msg
-card title linkName linkHref =
-    div [ css [ cardStyle ] ]
+
+-- There is currently only one quote in the copy, so a second paragraph is only displayed if one is added
+
+
+card : String -> String -> String -> String -> String -> Html msg
+card title quote1 description linkName linkHref =
+    div [ css cardStyle ]
         [ h2 []
             [ text title ]
-        , a
-            [ href linkHref ]
-            [ text linkName ]
+        , verticalSpacing
+        , blockquote [ css [ quoteStyle ] ]
+            [ p [] [ text quote1 ] ]
+
+        -- make this able to take a list
+        , verticalSpacing
+        , p [] [ text description ]
+        , verticalSpacing
+        , if String.length linkHref > 0 then
```

Comment:
This is probably a JS-y way to do this `link && <link-element>` - there could just be a different function if that's more normal here

Reply:
I would be tempted to output the Call to action in a separate function for each - and to avoid the 'Maybe' part of this, just a case for each card that outputs the exact html you want.